### DISCUSSION TITLE
Next/Image Placeholders (with BlurHash)

### DISCUSSION THREAD

**Discussion by joe-bell** [vercel/next.js#18382] [Show and tell]
Hey folks!

Huge fan of the latest image optimizations in Next.js 10 and wanted to add something a little "extra" to make the image loading experience a little more beautiful.

So I've built a little helper to generate **blurred placeholders** using BlurHash inside `getStaticProps`, introducing…

<h1><a href="https://github.com/joe-bell/next-blurhash">next-blurhash</a></h1>

It's super work in progress but I'd love to hear what you all think

**Comment by balazsorban44** [comment]
Nice, maybe something like that will even make it to the Image component itself! 🎉https://github.com/vercel/next.js/issues/18858
